flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -107,7 +107,7 @@ public void testRenameTable() {
         () -> getTableEnv().from("tl")
     );
     Assert.assertEquals(
-        Collections.singletonList(TableColumn.of("id", DataTypes.BIGINT())),
+        Collections.singletonList(TableColumn.physical("id", DataTypes.BIGINT())),
```

Comment:
ditto.

Reply:
Here,  we could use the similar way to avoid to use TableColumn#physical. https://github.com/openinx/incubator-iceberg/commit/2a635719955f3eecd92685cb8f5486c46aa60095#diff-4c56ab08c19464dbe3351f71fc39345ee031a282b3e8dc1b107cbe9a1964d105R102

Reply:
@stevenzwu ,  do you think  this issue need to fix in this PR ? I think we'd better to...